code/modules/mob/mob_defines.dm

```diff
@@ -107,6 +107,10 @@
 	/// Our body temperatue as of the last process, prevents pointless work when handling alerts
 	var/old_bodytemperature = 0
 	/// Drowsyness level of the mob
+	/// Dizziness level of the mob
+	var/dizziness = 0//Carbon
+	/// Jitteryness level of the mob
+	var/jitteriness = 0//Carbon
```

Comment:
These might not be necessary if /tg/ has removed them

Reply:
I can't find any other mobs/species to use that status effects and guessed it should be for status handling on IPC